common.sh

```diff
@@ -208,6 +214,12 @@ if [ -n "$MIRROR_IMAGES" ]; then
     export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="${LOCAL_REGISTRY_DNS_NAME}:${LOCAL_REGISTRY_PORT}/localimages/local-release-image:latest"
 fi
 
+if [[ ! -z "${OC_MIRROR}" ]] && [[ "${OC_MIRROR}" == "true" ]]; then
```

Comment:
Looks like this block should go before the previous one, ie in a similar fashion to when "_LOCAL_IMAGE" is defined (which implies activating `MIRROR_IMAGES` as well), otherwise `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE` will not be set

Reply:
Yep, good catch. Fixed.